rosbag2_snapshot/src/snapshotter.cpp

```diff
@@ -803,9 +839,18 @@ void Snapshotter::triggerSnapshotCb(
 
 void Snapshotter::clear()
 {
-  for (const buffers_t::value_type & pair : buffers_) {
-    pair.second->clear();
-  }
+  /*
```

Comment:
I think it would make more sense to comment out the code that uses the function instead of the function itself. One day someone else may need this and wait countless hours debugging wtf the function call does not work. is that possible?